src/speps.f

```diff
@@ -1,32 +1,33 @@
-C>@file
+C> @file
 C> @brief Compute utility spectral fields.
-C> @author IREDELL  @date 92-10-31
+C> @author Iredell  @date 92-10-31
 
 C> Computes constant fields indexed in the spectral domain
 C> in "IBM ORDER" (Zonal wavenumber is the slower index).
+C>      
 C> If L is the zonal wavenumber and N is the total wavenumber
 C> and A is the earth radius, then the fields returned are:
-C> - (1) NORMALIZING FACTOR EPSILON=SQRT((N**2-L**2)/(4*N**2-1))
-C> - (2) LAPLACIAN FACTOR N*(N+1)/A**2
-C> - (3) ZONAL DERIVATIVE/LAPLACIAN FACTOR L/(N*(N+1))*A
-C> - (4) MERIDIONAL DERIVATIVE/LAPLACIAN FACTOR EPSILON/N*A
+C> - (1) normalizing factor EPSILON=SQRT((N**2-L**2)/(4*N**2-1))
+C> - (2) Laplacian factor N*(N+1)/A**2
+C> - (3) zonal derivative/Laplacian factor L/(N*(N+1))*A
+C> - (4) Meridional derivative/Laplacian factor EPSILON/N*A
```

Comment:
I don't think "Meridional" needs to be capitalized. That said, I've been mostly been capitalizing variable descriptions and such (just the first word of a sentence/phrase). Do you prefer those to be decapitalized?

Reply:
I don't have strong opinions there but we are getting close to the end of this round of sp doxygen improvements, as far as I am concerned. We've made a lot of progress!